### ISSUE TITLE
java.lang.NoClassDefFoundError: Failed resolution of: Ljava/nio/file/Path;

### ISSUE THREAD

**Issue by mrtowel** [boonproject/boon#280]
When executing: 

`FakeDTO dataTransferObject = objectMapper.readValue(data, FakeDTO.class);`

We get: 
`Caused by: java.lang.ClassNotFoundException: Didn't find class "java.nio.file.Path" on path: DexPathList`

Environment: 
`FakeDTO` is a POJO with only one field (`boolean success`).
`objectMapper` is created via `JsonFactory.create()`.
`data` is a json string containing `{"success": true}`.

Execution runs is separate thread. 

Any help is appreciated.

**Comment by JosephEarl** [comment]
@vipinhelloindia @vsahu1986 I decided not to pursue the port of Boon JSON parser to Android for the time being for a couple of reasons:

* It's not clear Boon JSON parser is the fastest, the benchmarks only compare it to the main JSON parsers, GSON & Jackson. However a number of smaller JSON parsers exist which supposedly have better performance such as https://github.com/Instagram/ig-json-parser or https://github.com/Raizlabs/Parser 
* Lack of personal time

**Comment by RichardHightower** [comment]
I tested it against quite a few. There are many, many... and being the fastest is almost a self fulfilling prophecy of future loss unless I thought I was smarter than 9 billion other people which I do not. Also certain use cases (which I never run into) other parsers were faster for example when dealing with very large arrays of primitive types. That said... I compared it to the fastest parser I found and it did quite well which was a parser based out of Japan. It even does well against object serialization which are faster than Java object serialization. :) But it is good to see to new challengers.
